zebedee-cms/src/test/java/com/github/onsdigital/zebedee/api/ContentTest.java

```diff
@@ -118,22 +126,21 @@ public void writeVersionFileV1() throws Exception {
 
         Path versionFile = createCollectionAndPaths(collectionId);
 
-        // Create the version content via the content API.
+        // Given V1 Of the JSON was written
         try {
             Content contentApi = new Content();
             boolean result = contentApi.saveContent(request, new MockHttpServletResponse());
             Assert.assertTrue(result);
         } catch (Exception e) {
             e.printStackTrace();
-            Assert.fail(e.getMessage() + e.toString());
+            Assert.fail(e.getMessage() + e);
```

Comment:
This was actually also mentioned in the [original PR](https://github.com/ONSdigital/zebedee/pull/621#discussion_r868141374). I don't think we need this catch at all and we wouldn't need to be maintaining unnecessary code

Reply:
The test will not reporting a proper "failure message" if we leave it to throw an exception here, so was neccessary to catch.

Reply:
We do not expect an exception to occur. If it does, something has gone wrong and we should throw the exception. It is not a test failure but an error